src/pages/studyView/table/CNAGenesTable.tsx

```diff
@@ -88,14 +102,27 @@ export class CNAGenesTable extends React.Component<ICNAGenesTablePros, {}> {
         }
     }
 
+    toggleModal = (panelName: string, genes: GenePanelToGene[]) => {
```

Comment:
This should be a bound method.  You can do this with @autobind decorator.  Also, it should be have mobx's @action decorator.  Any time  we mutate mobx managed state we want to, want to do so inside of an action.  First and foremost, this batches the updates so you don't get multiple reactions (reactions being sycnronous).  It can also be useful for debugging because a debugger can log actions.  It's a little bit like redux's dispatcher.  Look or @action on other methods.

Reply:
Using the arrow syntax creates binding, my linter shows it as bound. I will add the @action one though!

Reply:
Ok I've updated it to this syntax